docs/src/highlevels/modeling_languages.md

```diff
@@ -40,51 +40,71 @@ For an overview of the library, see
 deterministic equations for how the mean and variance evolve in order to be solved in a single run. MomentClosure.jl
 uses Catalyst `ReactionSystem` and ModelingToolkit `SDESystem` types as the input for its symbolic generation
 processes.
 
+## Agents.jl: Agent-Based Modeling Framework in Julia
+
+If one wants to do agent-based modeling in Julia,
+[Agents.jl](https://docs.sciml.ai/Agents/stable/) is the go-to library. It's fast and flexible,
+making it a solid foundation for any agent-based model.
+
+## Unitful.jl: A Julia package for physical units
+
+Supports not only SI units but also any other unit system.
+[Unitful.jl](https://docs.sciml.ai/Unitful/stable/) has minimal run-time penalty of units.
+Includes facilities for dimensional analysis, and integrates easily with the usual mathematical operations and collections that are defined in Julia.
+
+## ReactionMechanismSimulator.jl: Simulation and Analysis of Large Chemical Reaction Systems
+
+[ReactionMechanismSimulator.jl](https://docs.sciml.ai/ReactionMechanismSimulator/stable/)
+is a tool for simulating and analyzing large chemical reaction mechanisms. It
+interfaces with the ReactionMechanismGenerator suite for automatically constructing reaction pathways
+from chemical components to quickly build realistic models of chemical systems.
+
 ## FiniteStateProjection.jl: Direct Solution of Chemical Master Equations
 
-[FiniteStateProjection.jl](https://github.com/kaandocal/FiniteStateProjection.jl) is a library for finite state
+[FiniteStateProjection.jl](https://docs.sciml.ai/FiniteStateProjection/dev/) is a library for finite state
 projection direct solving of the chemical master equation. It automatically converts the Catayst `ReactionSystem`
 definitions into ModelingToolkit `ODESystem` representations for the evolution of probability distributions to
 allow for directly solving the weak form of the stochastic model.
 
 ## AlgebraicPetri.jl: Applied Category Theory of Modeling
 
-[AlgebraicPetri.jl](https://github.com/AlgebraicJulia/AlgebraicPetri.jl) is a library for automating the intuitive
+[AlgebraicPetri.jl](https://docs.sciml.ai/AlgebraicPetri/stable/) is a library for automating the intuitive
 generation of dynamical models using a Category theory based approach.
 
-## Agents.jl: Agent-Based Modeling Framework in Julia
+## QuantumOptics.jl: Simulating quantum systems.
 
-If one wants to do agent-based modeling in Julia,
-[Agents.jl](https://github.com/JuliaDynamics/Agents.jl) is the go-to library. It's fast and flexible,
-making it a solid foundation for any agent-based model.
+[QuantumOptics.jl](https://docs.sciml.ai/QuantumOptics/stable/) makes it easy to simulate various kinds of quantum systems.
+It is inspired by the Quantum Optics Toolbox for MATLAB and the Python framework QuTiP.
+
+## MADS.jl: Model Analysis & Decision Support
+
+[MADS](https://github.com/madsjulia/Mads.jl)
+is an integrated high-performance computational framework for data/model/decision analyses.
```

Comment:
we should remove mads. It's not maintained.

Reply:
It looked still under active development to me.
https://github.com/madsjulia/Mads.jl/graphs/contributors

Reply:
ehh in some sense. I wouldn't recommend it though